namespaces/utils.go

```diff
@@ -37,8 +37,11 @@ func newInitPipe() (parent *os.File, child *os.File, err error) {
 
 // GetNamespaceFlags parses the container's Namespaces options to set the correct
 // flags on clone, unshare, and setns
-func GetNamespaceFlags(namespaces libcontainer.Namespaces) (flag int) {
+func GetNamespaceFlags(namespaces libcontainer.Namespaces, onlyNew bool) (flag int) {
```

Comment:
There is no usage for `onlyNew==false`, so I propose to remove it now and add it later if needed